encrypt/encrypt.go

```diff
@@ -206,13 +215,13 @@ func checkFiles(files []helpers.EncryptionFileSet) error {
 		}()
 
 		// Extracting the first 8 bytes of the header - crypt4gh
-		byteSlice := make([]byte, 8)
-		magicWord, err := unEncryptedFile.Read(byteSlice)
+		magicWord := make([]byte, 8)
+		_, err = unEncryptedFile.Read(magicWord)
 		if err != nil {
-			return err
+			return fmt.Errorf("error reading input file %s, reason: %v", file.Unencrypted, err)
 		}
-		if string(byteSlice[0:magicWord]) == "crypt4gh" {
-			return fmt.Errorf("Input file %s is already encrypted(.c4gh)", file.Unencrypted)
+		if string(magicWord) == "crypt4gh" {
+			return fmt.Errorf("input file %s is already encrypted(.c4gh) - make sure the right pk was used", file.Unencrypted)
```

Comment:
```suggestion
			return fmt.Errorf("input file %s is already encrypted(.c4gh)", file.Unencrypted)
```
If the file already is encrypted wrong input file was selected, not the wrong public key.

Reply:
That was meant to be a side note to the user, i.e. since we cannot check if this file was previously encrypted with the correct pk. But this looks confusing so I removed it.